### PR TITLE
Added [Term Entry] Swift Protocols Identifiable

### PR DESCRIPTION
**Pull request by AapoSoukkio**
### Description

- New term entry created for Swift [Protocols](https://www.codecademy.com/resources/docs/swift/protocols) about the Identifiable protocol. 
- New file has been created in docs/content/swift/concepts/protocols/terms/identifiable/identifiable.md
- Closes Issue #2883 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/swift/concepts/protocols/terms/identifiable/identifiable.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+Title: 'Identifiable'
+Description: 'Identifiable is a protocol in Swift that requires conforming types to have a unique identifier property'
```
Comment (SSwiniarski) [inline content/swift/concepts/protocols/terms/identifiable/identifiable.md:3]:
```suggestion
Description: 'A protocol in Swift that requires conforming types to have a unique identifier property.'
```

File: content/swift/concepts/protocols/terms/identifiable/identifiable.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+**`Identifiable`** is a protocol in Swift that enforces types to include a unique identifier property. This identifier helps differentiate individual instances of the type from one another within collections or arrays, making it easier to handle, update, and manage specific instances.
+
+## Syntax
+
+```pseudo
+protocol protocolName: Identifiable {
+  // Protocol body
+}
+```
+
```
Comment (SSwiniarski) [inline content/swift/concepts/protocols/terms/identifiable/identifiable.md:24]: We generally have a sentence or two in the syntax section summarizing what the syntax means.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@AapoSoukkio I did a first review and left a few comments.

File: content/swift/concepts/protocols/terms/identifiable/identifiable.md (hunk 2 above)
Comment (AapoSoukkio) [reply content/swift/concepts/protocols/terms/identifiable/identifiable.md:24]: "Indeed, I've noticed that now. It seems I overlooked that aspect

**Comment by AapoSoukkio** [comment]
@SSwiniarski  Thank you for your review. The entry has been updated as requested now.

**Review by SSwiniarski — APPROVED** [review]
@AapoSoukkio Looks ready for a second review.

File: content/swift/concepts/protocols/terms/identifiable/identifiable.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+**`Identifiable`** is a protocol in Swift that enforces types to include a unique identifier property. This identifier helps differentiate individual instances of the type from one another within collections or arrays, making it easier to handle, update, and manage specific instances.
+
+## Syntax
+
+An Identifiable protocol is created like regular protocol, followed by the `Identifiable` keyword:
```
Comment (caupolicandiaz) [inline content/swift/concepts/protocols/terms/identifiable/identifiable.md:19]:
```suggestion
An `Identifiable` protocol is created like a regular protocol, followed by the `Identifiable` keyword:
```

File: content/swift/concepts/protocols/terms/identifiable/identifiable.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+```pseudo
+protocol protocolName: Identifiable {
+  // Protocol body
+}
+```
+
+## Example
+
+Here, the Employee struct conforms to the Identifiable protocol by providing a unique `id` property using the `UUID` struct. This id can be used to distinguish instances of Employee in a collection or array, allowing for easy updates and deletions of specific employees.
```
Comment (caupolicandiaz) [inline content/swift/concepts/protocols/terms/identifiable/identifiable.md:29]:
```suggestion
Here, the `Employee` struct conforms to the `Identifiable` protocol by providing a unique `id` property using the `UUID` struct. This identifier can be used to distinguish instances of `Employee` in a collection or array, allowing for easy updates and deletions of specific employees.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@AapoSoukkio I've added a couple of suggestions. Please update, and tag me when you've finished making changes. Thanks

**Comment by AapoSoukkio** [comment]
@caupolicandiaz Thanks for the review! Code has been updated as requested.

**Review by caupolicandiaz — APPROVED** [review]
@AapoSoukkio @SSwiniarski LGTM

**Pull request merged**